.github/workflows/ci.yml

```diff
@@ -181,9 +175,14 @@ jobs:
           path: ${{ steps.cache-dir.outputs.dir }}
           key: ${{ runner.os }}-tests-${{ hashFiles('requirements/tests-requirements.txt') }}
 
+      - name: Generate gRPC stubs
+        run: |
+          pip install "grpcio-tools>=1.41" mypy-protobuf
+          python -m grpc_tools.protoc -I. --grpc_python_out=. --python_out=. --mypy_out=. --mypy_grpc_out=. bentoml/grpc/v1/service.proto bentoml/grpc/v1/service_test.proto
```

Comment:
Why do we need to run this manually? Does pip install not do this below?

Reply:
we don't install grpcio-tools by default. We have a separate container to run the generation to make sure the stubs are deterministic. For CI, we need this since Docker is only available on Linux CI, not Windows and MacOS.